### PR TITLE
accelerate the speed of making task

### PR DESCRIPTION
**Pull request by Angel-Jia**
Currently it takes about 1.2s to add a "tasks" with 101 POSCARs into the "stage".  The new code in line 187 and line 223 will make it 6 and 200 times faster, respectively

### CODE REVIEW

File: dpgen2/entrypoint/submit.py [hunk 1]
```diff
@@ -184,10 +184,11 @@ def make_conf_list(
         conf_list = []
         for ii in conf_list_fname:
             ss = dpdata.System(ii, type_map=type_map, fmt=fmt)
-            ss.to('lammps/lmp', 'tmp.lmp')
-            conf_list.append(Path('tmp.lmp').read_text())
-        if Path('tmp.lmp').is_file():
-            os.remove('tmp.lmp')
+            if fmt == 'vasp/poscar':
```
Comment (wanghan-iapcm) [inline dpgen2/entrypoint/submit.py:187]: Why do we need to check the `fmt`? It seems that other format other than `vasp/poscar' would work.

Comment (Angel-Jia) [reply dpgen2/entrypoint/submit.py:187]: Indeed, checking `fmt` is unnecessary, I had overlooked this.

File: dpgen2/entrypoint/submit.py [hunk 2]
```diff
@@ -236,7 +238,7 @@ def make_naive_exploration_scheduler(
                 sys_idx = jj.pop('conf_idx')
             conf_list = []        
             for ii in sys_idx:
-                conf_list += make_conf_list(sys_configs[ii], type_map)
+                conf_list.extend(sys_configs_lmp[ii])
```
Comment (wanghan-iapcm) [inline dpgen2/entrypoint/submit.py:241]: I do not see why the necessity of this change, it seems that the old and new code have almost the same behavior. Am I right?

Comment (Angel-Jia) [reply dpgen2/entrypoint/submit.py:241]: @wanghan-iapcm  Yes, they are almost the same, except that the new code is slightly faster when `conf_list`  is very large.

Comment (Angel-Jia) [reply dpgen2/entrypoint/submit.py:241]: @wanghan-iapcm I check it again and it turns out that I made a mistake. The code are almost the same as well as their performance even when `conf_list` is very large.

**Review by wanghan-iapcm — APPROVED** [review]

**Comment by wanghan-iapcm** [comment]
@Angel-Jia could you please check the failure in the UT? Is it due to bugs from dflow?

**Comment by Angel-Jia** [comment]
> @Angel-Jia could you please check the failure in the UT? Is it due to bugs from dflow?
OK

**Pull request merged**